core/src/main/java/org/apache/calcite/rex/RexBuilder.java

```diff
@@ -1110,6 +1110,14 @@ public RexLiteral makeApproxLiteral(BigDecimal bd) {
     return makeApproxLiteral(bd, typeFactory.createSqlType(SqlTypeName.DOUBLE));
   }
 
+  /**
```

Comment:
I think this method is too easy to invoke by accident.

Reply:
Yes, this should be deprecated. Can I do that?

Reply:
I have deleted the method